ledger_device_sdk/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ledger_device_sdk"
-version = "1.2.1"
+version = "1.3.0"
```

Comment:
You need to bump to `1.4.0` (it's already `1.3.0` on master)

Reply:
Done.